src/test_s2c_srv.c

```diff
@@ -382,6 +382,9 @@ int test_s2c(Connection *ctl, tcp_stat_agent *agent, TestOptions *testOptions,
           log_println(0, "S2C test error: can't create pipe.");
         } else {
           if ((s2c_childpid = fork()) == 0) {
+            // Don't capture more than 12 seconds of packet traces:
+            //   10 second test + 2 seconds of slop
+            alarm(12);
```

Comment:
The comments do explain it well. The named constants are less about clarity of the choices and more about clarity of the consistent behavior. If we're arbitrarily choosing `2` in identical scenarios, we might as well enforce the consistent, arbitrary choice in code. It's much easier to grep for `EXPECTED_TEST_DURATION_SECONDS` and see everywhere we use that constant than to grep for `10` in comments and pick through to figure out which `10`s refer to the same value.